src/index.js

```diff
@@ -107,6 +108,17 @@ async function cli () {
     error(`Output directory "${outputDir}/" doesn't exist`)
   }
 
+  if (!outputFormat) {
+    outputFormat = path.extname(output).replace('.', '')
+  }
+  if (outputFormat === 'md') {
+    // fallback to svg in case no outputFormat is given and output file is MD
+    outputFormat = 'svg'
+  }
+  if (!/(?:svg|png|pdf)$/.test(outputFormat)) {
+    error('Output format must be one of "svg", "png" or "pdf"')
+  }
```

Comment:
Can you move this into the `run()` function?

This library exposes the `run()` function, so we should move this default behavior into that function to maintain backwards compatibility (even if it's a bit slower).

We'd also need to change the error condition to `throw new Error('Output format must be one of "svg", "png" or "pdf"');`

Reply:
Fixed in https://github.com/mermaid-js/mermaid-cli/pull/382/commits/fa79772dedda70470830ad2af17b0e25b5c89a7a and https://github.com/mermaid-js/mermaid-cli/pull/382/commits/c36859c2cf393e7b5d0cf8b5007c7bdee9e759ee